### DISCUSSION TITLE
自定义试炼刷怪笼cd

### DISCUSSION THREAD

**Discussion by wendavid552** [Minecraft-AMS/Carpet-AMS-Addition#131] [Ideas]
### Motivation

试炼刷怪笼生成奖励之后会进入36000游戏刻（30分）的冷却状态，亮度等级转为0，普通试炼刷怪笼顶部的橙色边角变灰、不祥试炼刷怪笼顶部的蓝色边角变黑且内部不再显示生物模型和火焰粒子。冷却结束后才能再次激活。

### Description

自定义试炼刷怪笼的cd，例如调0

**Comment by 1024-byteeeee** [comment]
我们再等几个mc版本出了之后再做新内容的规则？

**Comment by wendavid552** [comment]
> 我们再等几个mc版本出了之后再做新内容的规则？

其实主要是Carpet-TIS出了个可以无限刷的，所以对应地做这样一个功能。其实现在有很多遗留问题待解决，比如一些功能和其他Mod的兼容之类的

**Comment by liuyuexiaoyu1** [comment]
https://github.com/liuyuexiaoyu1/Carpet-Igny-Addition/commit/7f65b3e2f74ce7bde37c76e89660d8fc4d682ff4，也许算是实现了？（不确定有没有写对😭）
